leapp/cli/upgrade/__init__.py

```diff
@@ -149,6 +150,11 @@ def upgrade(args):
         logger = configure_logger('leapp-upgrade.log')
     os.environ['LEAPP_EXECUTION_ID'] = context
 
+    until_phase = None
+    if args.pre_upgrade:
+        until_phase = "DownloadPhase"
```

Comment:
I propose to set the dedault to the report phase. As initial transaction checks will be part of the check phase. Additionally, there would be additional option like `--include-download` that would continue with shift the end until the download phase.

Reply:
This is a good topic to discuss at Cabal, since others already gave me (conflicting) opinions on this. Adding to agenda.

Reply:
(Just for context's sake, the solution we've decided to use is the one proposed here.)